sideboard/tests/test_configuration.py

```diff
@@ -33,7 +49,114 @@ def test_parse_config_adding_root_and_module_root_for_production(self):
             assert test_config.get('root') == expected_root
 
 
-class TestSideboardConfigRoot(object):
+class TestSideboardGetConfigFiles(object):
+    @pytest.fixture
+    def config_overrides_unset(self, monkeypatch):
+        monkeypatch.delenv('SIDEBOARD_CONFIG_OVERRIDES', raising=False)
+
+    @pytest.fixture
+    def config_overrides_set(self, monkeypatch):
+        monkeypatch.setenv('SIDEBOARD_CONFIG_OVERRIDES', 'test-defaults.ini')
+
+    @pytest.fixture
+    def plugin_dirs(self):
+        module_path = '/opt/sideboard/plugins/test_plugin/test_plugin'
+        root_path = os.path.join(os.getcwd(), 'plugins', 'test_plugin')
+        return (module_path, root_path)
+
+    @pytest.fixture
+    def sideboard_dirs(self):
+        module_path = '/opt/sideboard/sideboard'
+        root_path = '/opt/sideboard'
+        return (module_path, root_path)
+
+    def test_get_module_and_root_dirs_plugin(self, plugin_dirs):
```

Comment:
Our use of ``os.path.join`` and ``os.path.sep`` is somewhat vestigial.  Originally when we were coding Sideboard, we though, "We should use ``os.path.join`` on general principle in case we ever need to run this on Windows or something crazy like that."  Fast forward to now and I think it's safe to say that Sideboard is definitely a Unix-based system and we simply don't ever intend to support it on any other platform.  I still tend to use things like ``os.path.join`` anyway just out of habit and good general practice, but it's definitely not a problem to assume ``/`` as a path separator IMO.